main/src/ca/uwaterloo/flix/language/ast/Scheme.scala

```diff
@@ -89,14 +90,14 @@ object Scheme {
     * Returns `true` if the given schemes are equivalent.
     */
   // TODO can optimize?
-  def equal(sc1: Scheme, sc2: Scheme, classEnv: Map[Symbol.ClassSym, Ast.ClassContext])(implicit flix: Flix): Boolean = {
+  def equal(sc1: Scheme, sc2: Scheme, classEnv: Map[Symbol.ClassSym, Ast.ClassContext])(implicit univ: Ast.Multiverse, flix: Flix): Boolean = {
```

Comment:
Why not just pass `root`?

Reply:
I would much prefer if we passed `root` as much as possible.

Reply:
I've now changed to do that inside of phases, but places like `Scheme` and `unification` are phase-ambivalent so we can't pass root there.